test/unit/SandboxCoordinatorSpec.ts

```diff
@@ -73,6 +78,67 @@ describe('SandboxCoordinator', () => {
     it('should resolve in expected result', () => expect(actualRunResult).to.be.eq(expectedRunResult));
   });
 
+  describe('Given that maxConcurrentTestRunners config has been set', () => {
+    it('if config is less than cpuCount, then runMutants should use that config rather than cpuCount', (done) => {
+      const mutants: any[] = [mockMutant(0, []), mockMutant(1, [])];
+
+      sut = new SandboxCoordinator({
+        maxConcurrentTestRunners: 1
+      }, expectedInputFiles, expectedTestFramework, reporter);
+
+      firstSandbox.runMutant.returns(Promise.resolve({ status: RunStatus.Complete, tests: [{ name: 'test1', status: TestStatus.Success }] }));
+      secondSandbox.runMutant.returns(Promise.resolve({ status: RunStatus.Complete, tests: [{ name: 'test1', status: TestStatus.Success }] }));
+
+      sut.runMutants(mutants)
+        .then(() =>  {
+          expect(sandbox.default).to.have.been.calledWithNew;
+          expect(sandbox.default).to.have.callCount(1);
+          expect(sandbox.default).to.have.been.calledWith({ maxConcurrentTestRunners: 1 }, 0, expectedInputFiles, expectedTestFramework, null);
+          done();
+        });
+    });
+
+    it('if config is greater than cpuCount, then runMutants should use the cpuCount', (done) => {
```

Comment:
have changed a bit. I think it's fine
![image](https://cloud.githubusercontent.com/assets/1034890/20638902/a1042ae0-b408-11e6-8607-95e8158bc304.png)